scipy/linalg/setup.py

```diff
@@ -119,6 +119,31 @@ def configuration(parent_package='',top_path=None):
 
     config.add_data_dir('tests')
     config.add_data_dir('benchmarks')
+
+    config.add_data_files('blas_pointers.pxd')
+    config.add_data_files('lapack_pointers.pxd')
+
+    config.add_data_files('f2pyptr.h')
+
+    config.add_extension('blas_pointers',
+                         sources=['blas_pointers.c'],
+                         include_dirs=['.'])
+
+    config.add_extension('lapack_pointers',
+                         sources=['lapack_pointers.c'],
+                         include_dirs=['.'])
+
+    config.add_extension('_test_blas_pointers',
+                         sources=['_test_blas_pointers.c'],
+                         include_dirs=['.'])
+
+    config.add_extension('_test_lapack_pointers',
+                         sources=['_test_lapack_pointers.c'],
+                         include_dirs=['.'])
+
+    config.add_extension('_decomp_update',
+                         sources=['_decomp_update.c'])
```

Comment:
The lines to add in ``linalg/bento.info`` are:

    Extension: _decomp_update
        Sources:
            _decomp_update.c